### ISSUE TITLE
asset 16 in sprint 6 set 

### ISSUE THREAD

**Issue by colleenXu** [NCATSTranslator/Tests#109]
@maximusunc 

Asset 16 was previously removed for bad curie / duplicate reasons. https://github.com/NCATSTranslator/Tests/issues/50 

However, I see it in the sprint 5/6 runs. Should this asset be removed from those runs?

**Comment by vdancik** [comment]
I agree, asset 16 should be removed, since CHEBI:50454 has not been annotated and is a preliminary ChEBI entity.

**Comment by maximusunc** [comment]
Removed in [cdbe8b4](https://github.com/NCATSTranslator/Tests/pull/114/commits/cdbe8b465f9a1225374d04fb565ed0913a476ac5)
